frontend/package-lock.json

```diff
@@ -1,8 +1,16112 @@
 {
     "name": "ab-testing",
     "version": "1.0.52",
-    "lockfileVersion": 1,
+    "lockfileVersion": 2,
```

Comment:
@VivekFitkariwala Do we really need this change as we don't have any changes in the package.json file?

Reply:
This is due to different npm version we are using. I think if we are npm > ^7.0.0 it uses `"lockfileVersion": 2`. I have reverted the change for now but I think we can force user to use minimum version of npm to avoid this conflict in future